save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityDateModal.kt

```diff
@@ -25,17 +25,20 @@ import kotlinx.serialization.json.Json
  */
 val vulnerabilityDateModal: FC<VulnerabilityDateModalProps> = FC { props ->
     val (vulnerabilityDate, setVulnerabilityDate) = useState(VulnerabilityDateDto.empty)
-
+    val (conflictMessage, setConflictMessage) = useState<String?>(null)
```

Comment:
as I remember, you do prefer `useState<T?>(null)` instead of `useState<T>()`, don't you?
In case it's not -- `useState<T>() == useState<T?>(null)`

Reply:
I know it is basically the same (except `useState<T>()` __might__ be `undefined`), but yes, I do prefer the first variant